pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -427,6 +436,37 @@ func GatherClusterOperators(i *Gatherer) func() ([]record.Record, []error) {
 	}
 }
 
+func collectClusterOperatorResources(i *Gatherer, co configv1.ClusterOperator) *clusterOperatorResource {
+	var relObj configv1.ObjectReference
+	for _, ro := range co.Status.RelatedObjects {
+		if ro.Group == "operator.openshift.io" {
+			relObj = ro
+		}
+	}
+	if relObj == (configv1.ObjectReference{}) {
+		return nil
+	}
+	gvr := schema.GroupVersionResource{Group: "operator.openshift.io", Version: "v1", Resource: strings.ToLower(relObj.Resource)}
+	clusterResource, err := i.dynamicClient.Resource(gvr).Get(i.ctx, "cluster", metav1.GetOptions{})
```

Comment:
Instead of "cluster" should be name of related resource I think (even thought they are all cluster in our case), and I would like a bit more if Group would be also from relObj (even though its semantically the same).